dev/cnf/oss_dependencies.maven

```diff
@@ -396,7 +396,7 @@ org.quartz-scheduler:quartz:2.2.1
 org.quartz-scheduler:quartz:2.2.1
 org.reactivestreams:reactive-streams-examples:1.0.2
 org.reactivestreams:reactive-streams:1.0.2
-org.reactivestreams:reactive-streams:1.0.2
```

Comment:
Hmmm, surprised that I missed that duplicate line when I tidied this file up.

Reply:
Yeah, so was I